OsmAnd/res/values/strings.xml

```diff
@@ -4293,8 +4293,8 @@ Download tile maps directly, or copy them as SQLite database files to OsmAnd\'s
     <string name="search_offline_clear_search">New Search</string>
     <string name="map_text_size_descr">Text size for names on the map:</string>
     <string name="map_text_size">Map font size</string>
-    <string name="trace_rendering">Rendering debug info</string>
-    <string name="trace_rendering_descr">Display the rendering performance.</string>
+    <string name="trace_rendering">Performance debug info</string>
```

Comment:
I don't think now rendering performance is used, it was very annoying toast on each render frame where navigation toast is just once per calculation

Reply:
Well, both are "annoying" unless you specifically need them to debug something specific.  ;)

Hence I think they should be "opt-in", even when the Dev plugin is active, to avoid further user confusion and more issues opened about it.

So the question becomes: _Can the debug toasts be productively used for debugging if there is one combined switch to activate / deactivate both in sync (like this PR would do)?_ I have tested that things work ok, no toast is lost, and should there are parallel toasts for rendering and route calculation (which there often are), they show in succession. So it could really be a viable approach, feel free to test.